Added formik with Yup and implemented in login

ketanp-techuz opened the pull request.

ashishtechuz commented:
Please fix eslint warnings to let deployment happen. Any warning while generating a build will fail the deployment.

ketanp-techuz commented:
> Please fix eslint warnings to let deployment happen. Any warning while generating a build will fail the deployment.

Fixed, warning because of it's not sync with master.

ashishtechuz commented on src/pages/Login/Login.jsx:
```diff
@@ -1,44 +1,59 @@
+    },
+    validationSchema: validationSchema,
+    onSubmit: (values) => {
+      console.log("values", values);
```
Please remove console log after testing.

ketanp-techuz commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Added formik with Yup and implemented in login](https://trello.com/c/qvvZsanP/6-added-formik-with-yup-and-implemented-in-login)
